packages/app-file-manager/src/admin/components/FileManagerSettings.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import { Form } from "@webiny/form";
+import { Grid, Cell } from "@webiny/ui/Grid";
+import { Switch } from "@webiny/ui/Switch";
+import { ButtonPrimary } from "@webiny/ui/Button";
+import { Query, Mutation } from "react-apollo";
+import { useSnackbar } from "@webiny/app-admin/hooks/useSnackbar";
+import { Input } from "@webiny/ui/Input";
+import graphql from "../graphql";
+import { CircularProgress } from "@webiny/ui/Progress";
+import get from "lodash.get";
+
+import {
+    SimpleForm,
+    SimpleFormFooter,
+    SimpleFormContent,
+    SimpleFormHeader
+} from "@webiny/app-admin/components/SimpleForm";
+
+const FileManagerSettings = () => {
+    const { showSnackbar } = useSnackbar();
+    return (
+        <Query query={graphql.query}>
+            {({ data, loading: queryInProgress }) => (
+                <Mutation mutation={graphql.mutation}>
+                    {(update, { loading: mutationInProgress }) => {
+                        const settings = get(data, "files.getSettings.data") || {};
+
+                        return (
+                            <Form
+                                data={settings}
+                                onSubmit={async data => {
+                                    await update({
+                                        variables: {
+                                            data: {
+                                                uploadMinFileSize: parseInt(data.uploadMinFileSize),
+                                                uploadMaxFileSize: parseInt(data.uploadMaxFileSize)
+                                            }
+                                        }
+                                    });
+                                    showSnackbar("Settings updated successfully.");
+                                }}
+                            >
+                                {({ Bind, form, data }) => (
+                                    <SimpleForm>
+                                        {(queryInProgress || mutationInProgress) && (
+                                            <CircularProgress />
+                                        )}
+                                        <SimpleFormHeader title="File Manager Settings" />
+                                        <SimpleFormContent>
+                                            <Grid>
+                                                <Cell span={12}>
+                                                    <Grid>
+                                                        <Cell span={6}>
+                                                            <Bind name={"uploadMinFileSize"}>
+                                                                <Input
+                                                                    type="number"
+                                                                    label="Minimum File Upload Size (Bytes)"
```

Comment:
Please follow the same style we had in other settings for labels. In other words, no need for uppercasing every word in the label. For example:

![image](https://user-images.githubusercontent.com/5121148/84020777-a9a20900-a983-11ea-810e-afabea1f1c86.png)

Also, please move the (Bytes) part into the `description` prop, and let's use something like `In bytes.`.